src/renderer/angular/directives/commons/sk-circle-pie-chart.directive.js

```diff
@@ -64,14 +64,15 @@ function SkCirclePieChartDirective($timeout, CommonService) {
                         subTitle: '',
                         isOtherItem: true,
                         valueUSD: new BigNumber(0),
+                        unformattedValueUSD: new BigNumber(0),
                         color: colorForOthers
                     };
 
                     otherItems.forEach(otherItem => {
-                        otherAggregated.valueUSD = new BigNumber(otherAggregated.valueUSD).plus(new BigNumber(otherItem.valueUSD));
+                        otherAggregated.unformattedValueUSD = new BigNumber(otherAggregated.unformattedValueUSD).plus(new BigNumber(otherItem.unformattedValueUSD));
                     });
 
-                    otherAggregated.valueUSD = +CommonService.numbersAfterComma(otherAggregated.valueUSD, 2);
+                    otherAggregated.valueUSD = +Intl.NumberFormat('en-US').format(Number(otherAggregated.unformattedValueUSD).toFixed(2));
```

Comment:
Passing USD attributes around is going to make implementing localized currencies hell.

We should pass around a standard `fiatValue` attribute which contains a number which may represent any fiat currency.